js/iframe-coordinator/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "module": "commonjs",
     "declaration": true,
     "declarationDir": "./",
-    "target": "es5"
+    "target": "ES6",
```

Comment:
I bumped this up to allow the usage of some functional array methods.

@MattCheely we probably want to distribute es6 modules (ts and/or js) and es5 UMD modules for maximum consumability and tooling, correct?

Reply:
Are there still places where people don't transpile?  It would be nice if we could only distrubute es6.